src/server/worker/datum/datum.go

```diff
@@ -1,3 +1,19 @@
+/*
+Package datum handles datums, the core units of work in a pipeline.
+
+We use a specific file system structure to store datums, whether in meta commits or in temporary lists during job processing.
+Every datum has a JSON-marshalled [datum.Meta] at the file path
+
+	/meta/<datum-ID>/meta
+
+When computing a job's list of datums, the ID is prefixed with an index so that datums are in a cache-friendly order.
```

Comment:
what does the prefixed ID look like? Could you point me to that code?

Reply:
It is changing a bit with the PR I am working on. It is just an integer that corresponds with the order the file was produced by glob file when creating the datums for the PFS input.

Reply:
Added a pointer in the docs, as well as some PFS stuff, but it's this bit in getFileSet
https://github.com/pachyderm/pachyderm/blob/4362569599f9ed2b3d38738dbf3a6ff8b032c71e/src/server/pfs/server/driver_file.go#L430-L452

Reply:
oooh that embedded code link is pretty nifty

Reply:
ha yeah, I tried to be vague enough to accurately capture the things that were changing